config.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget id="io.ionic.starter" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+    <name>MyApp</name>
+    <description>An awesome Ionic/Cordova app.</description>
```

Comment:
fix these things as per project details 

Name: ReOrder App
Description: App to quickly reorder items
auth: Ronn@ron.com